st2common/st2common/config.py

```diff
@@ -62,7 +62,9 @@ def register_opts(ignore_errors=False):
 
     system_opts = [
         cfg.StrOpt('base_path', default='/opt/stackstorm',
-                   help='Base path to all st2 artifacts.')
+                   help='Base path to all st2 artifacts.'),
+        cfg.ListOpt('admin_users', default=None,
```

Comment:
Good catch - defaulting to an empty list here would actually be fine since we don't mutate it later.